### ISSUE TITLE
blutooth is not working

### ISSUE THREAD

**Issue by ae-circuit** [Sycnex/Windows10Debloater#344]
After debloating the windows the option to turn on/off Bluetooth has gone, please help

**Comment by Gray-0men** [comment]
Right click task bar > Taskbar settings > Turn System Icons on or off

**Comment by ae-circuit** [comment]
No, if that was the problem I would have solved it. The bluetooth is not detected on the system. Its working on linux only

**Comment by IamVSJ** [comment]
No @Gray-0men I would have fixed this simple issue.
its not detecting bluetooth on my PC, but its working perfect on (dual boot) linux.

**Comment by Sycnex** [comment]
@IamVSJ - It could be that you need to reinstall the bluetooth driver, but I am not certain. I would look at reinstalling the drivers from your manufacturer's website for your computer.
